docs/graphql/manual/remote-schemas/index.rst

```diff
@@ -4,12 +4,12 @@
 
 .. _remote_schemas:
 
-Remote schemas
+Remote Schemas
```

Comment:
A table of contents is here but can we have a section called "Introduction" introducing "Remote schemas"?